gsplat/rasterize.py

```diff
@@ -231,6 +233,11 @@ def backward(ctx, v_out_img, v_out_alpha=None):
                 v_out_alpha,
             )
 
+        # Abs grad for gaussian splitting criterion. See
+        # - "AbsGS: Recovering Fine Details for 3D Gaussian Splatting"
+        # - "EfficientGS: Streamlining Gaussian Splatting for Large-Scale High-Resolution Scene Representation"
+        xys.absgrad = v_xy_abs
```

Comment:
I wonder if this will cause problems in splat training? currently there's a `xys.retain_grad()` in the nerfstudio implementation that's needed to keep the `.grad` property around. We should verify that this absgrad property works in the nerfstudio training pipeline

Reply:
The only issue of this hack is it doesn't do accumulation like xys.grad if multiple backward() calls are triggered. Let's say if one rasterize gaussians twice in the forward call, the backward would be called twice, but only the latest v_xy_abs is saved. Maybe add a comment here to indicate the difference in how backward call would be updating xys.grad and xys.absgrad.